src/test/java/seedu/address/model/note/TagNoteMapTest.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.model.note;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalClients.ALICE;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.client.Client;
+import seedu.address.model.tag.Tag;
+
+class TagNoteMapTest {
+
+    private static final Tag TAG = new Tag("tagName");
```

Comment:
Not sure why only this variable is static? Could probably make all of them just `final`.